TTHAnalysis/python/plotter/susy-sos/sos_plots.py

```diff
@@ -220,7 +225,7 @@ def prepareWrapper(name):
         f.write( 'echo ${CARDS_ALL}\n' )
         f.write( 'cd %s\n'%(HIGGSCOMBINEDIR) )
         f.write( 'eval `scramv1 runtime -sh`\n' )
-        f.write( 'cd -\n' )
+        f.write( 'cd %s\n'%(ODIR) )
         f.write( '[ -d %s/combinedCards ] || mkdir %s/combinedCards\n'%(ODIR,ODIR) )
```

Comment:
If in directory '$ODIR', does it make sense to mkdir '$ODIR/combinedCards' ? Does this change from cd-ing into '$ODIR/..' to '$ODIR' ?  I am not positive. Can test and report back...

Reply:
Thanks for reporting this, I will fix it.